donate/routes.py

```diff
@@ -143,8 +143,10 @@ def donation():
             params['email'])
         app.logger.debug("Charge created: {}".format(charge_data))
     except StripeError as error:
```

Comment:
using `StripeError` as a catch all without looking more carefully at what type of error it is is contrary to the Stripe api documentation's reccomendations and likely to result in these sorts of bugs in the future; the [handling errors](https://stripe.com/docs/api/errors/handling) Stripe docs layout what a better alternative looks like.